tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -162,6 +169,15 @@
 
     window.test_driver_internal.in_automation = true;
 
+    window.test_driver_bidi_internal.session.subscribe = function(event, context=null) {
+        return create_action("subscribe", {event, context});
```

Comment:
we should probably prefix all actions to avoid clashes with classic actions?

Reply:
e.g., `bidi.session.subscribe`

Reply:
the `test_driver_internal` is not exposed to user, so there should be no clashes. But it can be moved under `test_driver_internal.bidi`. Which is actually already done.

Reply:
I mean in `create_action("subscribe", {event, context});`

Reply:
fixed